charts/umbrella/Chart.yaml

```diff
@@ -28,7 +28,7 @@ sources:
   - https://github.com/eclipse-tractusx/e2e-testing
 
 type: application
-version: 0.3.0
+version: 0.3.1
```

Comment:
As discussed, I'd bump the version to minor for adding a new product.

Reply:
done